src/general_manager/bucket/calculationBucket.py

```
@@ -157,29 +197,77 @@ def __repr__(self) -> str:
     def filter(self, **kwargs: Any) -> CalculationBucket:
         """
         Returns a new CalculationBucket with additional filters applied.
 
         Merges the provided filter criteria with existing filters to further restrict valid input combinations.
         """
-        filters = self.filters.copy()
-        excludes = self.excludes.copy()
-        filters.update(parse_filters(kwargs, self.input_fields))
-        return CalculationBucket(self._manager_class, filters, excludes)
+        properties = self._manager_class.Interface.getGraphQLProperties()
+        possible_values = self.transformPropertiesToInputFields(
```

Ruff (0.12.2)

Local variable `possible_values` is assigned to but never used

Remove assignment to unused variable `possible_values`

(F841)